services/backend/src/persistence/protected_area.js

```diff
@@ -313,5 +313,53 @@ module.exports = (db, { globalBinaryProtectedAreas }) => {
         )
         .then((layers) => layers.rows[0].collection);
     },
+
+    /**
+     * Get the coverage layer divided by categories in a given protected area category
+     * @param {String} categoryName protected area category
+     *
+     * @return {Object} Geojson object with the geometry
+     */
+    findCoverageLayer: async (categoryName) => {
+      let bitMask = await globalBinaryProtectedAreas
+        .query()
+        .where({ label: categoryName })
+        .select('binary_protected as mask');
+      bitMask = bitMask[0].mask;
+      return db
+        .raw(
+          `
```

Comment:
this indentation (in all persistences) seems odd, I think the query should be one level deeper than the opening `` ` ``, or at least the same level

Reply:
Done